src/ord.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const {tagged} = require('daggy');
+const {empty, of, concat, equals} = require('fantasy-land');
+
+const Ord = M => {
+
+    const _Ord = tagged('x');
+
+    _Ord[of] = _Ord;
+    _Ord[Ord.min] = M[Ord.min];
+    _Ord[Ord.max] = M[Ord.max];
+
+    _Ord.prototype.compare = function(y) {
+      return M.compare(this.x, y.x);
+    };
+    _Ord.prototype[equals] = function(y) {
```

Comment:
Why `prototype[equals]` instead of just `prototype.equals` ? just looks weird to me ;)